### ISSUE TITLE
Hello skill not working

### ISSUE THREAD

**Issue by mouryachintalapati** [opsdroid/opsdroid#1310]
Hi,

I have setup opsdroid in windows 10

Versions: 
Opsdroid 0.17.0
Python 3.8.1

Configuration.yaml: 

[configuration.txt](https://github.com/opsdroid/opsdroid/files/3991251/configuration.txt)

Right after launching opsdroid with start command: when i type "hi" in the windows client seeing below error: 

```

C:\Users\integrat>opsdroid start
INFO opsdroid.logging: ========================================
INFO opsdroid.logging: Started opsdroid v0.17.0.
INFO opsdroid: ========================================
INFO opsdroid: You can customise your opsdroid by modifying your configuration.yaml.
INFO opsdroid: Read more at: http://opsdroid.readthedocs.io/#configuration
INFO opsdroid: Watch the Get Started Videos at: http://bit.ly/2fnC0Fh
INFO opsdroid: Install Opsdroid Desktop at:
https://github.com/opsdroid/opsdroid-desktop/releases
INFO opsdroid: ========================================
WARNING opsdroid.loader: No databases in configuration. This will cause skills which store things in memory to lose data when opsdroid is restarted.
INFO opsdroid.web: Started web server on http://0.0.0.0:8080
INFO opsdroid.core: Opsdroid is now running, press ctrl+c to exit.
INFO aiohttp.access: 10.52.81.154 [21/Dec/2019:09:39:20 +0000] "POST /connector/websocket HTTP/1.1" 200 252 "-" "-"
ERROR opsdroid.core: Exception when running skill 'hello'.
Traceback (most recent call last):
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\opsdroid\core.py", line 396, in run_skill
    return await skill(event)
  File "C:\Users\integrat\AppData\Local\opsdroid-modules\opsdroid\opsdroid-modules\skill\hello\__init__.py", line 13, in hello
    await message.respond(text)
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\opsdroid\events.py", line 263, in respond
    "thinking-delay" in self.connector.configuration
AttributeError: 'NoneType' object has no attribute 'configuration'
ERROR aiohttp.server: Error handling request
Traceback (most recent call last):
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\opsdroid\core.py", line 396, in run_skill
    return await skill(event)
  File "C:\Users\integrat\AppData\Local\opsdroid-modules\opsdroid\opsdroid-modules\skill\hello\__init__.py", line 13, in hello
    await message.respond(text)
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\opsdroid\events.py", line 263, in respond
    "thinking-delay" in self.connector.configuration
AttributeError: 'NoneType' object has no attribute 'configuration'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\aiohttp\web_protocol.py", line 418, in start
    resp = await task
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\aiohttp\web_app.py", line 458, in _handle
    resp = await handler(request)
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\opsdroid\connector\websocket\__init__.py", line 101, in websocket_handler
    await self.opsdroid.parse(message)
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\opsdroid\core.py", line 509, in parse
    await asyncio.gather(*tasks)
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\opsdroid\core.py", line 402, in run_skill
    await event.respond(
  File "c:\users\integrat\appdata\local\programs\python\python38-32\lib\site-packages\opsdroid\events.py", line 263, in respond
    "thinking-delay" in self.connector.configuration
AttributeError: 'NoneType' object has no attribute 'configuration'
INFO aiohttp.access: 10.52.81.154 [21/Dec/2019:09:39:24 +0000] "POST /connector/websocket HTTP/1.1" 200 252 "-" "-"
`
```

**Comment by jacobtomlinson** [comment]
Sorry for the delay in getting back to you! Looks like this is a bug and was missed in #1116.

**Comment by jacobtomlinson** [comment]
This is now fixed in v0.17.1. Please upgrade your opsdroid.
